README.md

```diff
@@ -114,47 +107,45 @@ If the decline button is enabled then the (onDecline) prop function can be used,
 
 ## Props
 
-| Prop                     |               Type                | Default value                        | Description                                                                                                                             |
```

Comment:
Please revert the layout changes

Reply:
The layout changed because of the git hook (prettier) since the column now is shorter than before